src/views/vesting/Withdraw.svelte

```diff
@@ -0,0 +1,72 @@
+<script lang="ts" strictEvents>
+  import type { VestingInfo } from "@app/lib/vesting";
+  import type { Wallet } from "@app/lib/wallet";
+
+  import * as utils from "@app/lib/utils";
+  import Button from "@app/components/Button.svelte";
+  import ErrorModal from "@app/components/ErrorModal.svelte";
+  import Loading from "@app/components/Loading.svelte";
+  import Modal from "@app/components/Modal.svelte";
+  import { createEventDispatcher, onMount } from "svelte";
+  import { state } from "@app/lib/vesting";
+  import { withdrawVested } from "@app/lib/vesting";
+
+  export let contractAddress: string;
+  export let info: VestingInfo;
+  export let wallet: Wallet;
+
+  const dispatch = createEventDispatcher<{ close: never }>();
+
+  onMount(async () => {
+    await withdrawVested(contractAddress, wallet);
+  });
+</script>
+
+<style>
+  .actions {
+    display: flex;
+    justify-content: center;
+    flex-direction: row;
+    gap: 1rem;
+  }
+</style>
+
+{#if $state.type === "error"}
+  <ErrorModal
+    floating
+    title="Withdraw failed"
+    message={$state.error}
+    on:close={() => dispatch("close")} />
+{:else}
+  <Modal on:close floating>
+    <span slot="title">
+      {utils.formatAddress(contractAddress)}
+    </span>
+
+    <span slot="subtitle">
+      {#if $state.type === "withdrawingSign"}
+        <span class="txt-missing">Waiting for a signature...</span>
```

Comment:
Could we use the ellipsis character … instead of `...` everywhere?

Reply:
5d2cbd3